compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/UIKitComposeSceneLayer.uikit.kt

```diff
@@ -115,22 +113,17 @@ internal class UIKitComposeSceneLayer(
 
     fun render(canvas: Canvas, nanoTime: Long) {
         if (scrimColor != null) {
-            val size = view.bounds.useContents { with(density) { size.asDpSize().toSize() } }
-
             canvas.drawRect(
                 left = 0f,
                 top = 0f,
-                right = size.width,
-                bottom = size.height,
+                right = Float.MAX_VALUE,
+                bottom = Float.MAX_VALUE,
```

Comment:
Why was it changed? Did it affect the performance?

Reply:
During the animation, canvas can be greater than layer's view. And because of aync scene updates. schirm may have glitches on edges.
This solution does not degrade performance, but I found another one with getting bounds of MetalView, PTAL.